interpreter/cling/lib/Interpreter/ASTImportSource.cpp

```diff
@@ -168,4 +201,55 @@ namespace cling {
       }
       return false;
     }
+
+    ///\brief Make available to child all decls in parent's decl context
+    /// that corresponds to child decl context.
+    void ASTImportSource::completeVisibleDeclsMap(
+                                  const clang::DeclContext *childDeclContext) {
+      assert (childDeclContext && "No child decl context!");
+
+      if (!childDeclContext->hasExternalVisibleStorage())
+        return;
+
+      // Search in the map of the stored Decl Contexts for this
+      // Decl Context.
+      std::map<const clang::DeclContext *, clang::DeclContext *>::iterator I;
+      // If childCurrentDeclContext was found before and is already in the map,
+      // then do the lookup using the stored pointer.
+      if ((I = m_DeclContexts_map.find(childDeclContext))
+          != m_DeclContexts_map.end()) {
```

Comment:
I'd prefer an early return (if == end) instead of a long block of indented code.